logprep/framework/pipeline.py

```diff
@@ -436,9 +436,10 @@ def process_event(self, event: dict):
                     self._handle_processing_warning(processor, warning)
             except BaseException as error:  # pylint: disable=broad-except
                 msg = self._handle_fatal_processing_error(processor, error)
-                for _, output in self._output.items():
-                    if output.default:
-                        output.store_failed(msg, json.loads(event_received), event)
+                if self._output:
+                    for _, output in self._output.items():
+                        if output.default:
+                            output.store_failed(msg, json.loads(event_received), event)
```

Comment:
could you remove this change? it leads to merge conflicts in #344

Reply:
well I can't because then I have a failed test:
```bash
>               for _, output in self._output.items():
E               AttributeError: 'NoneType' object has no attribute 'items'
```